main.go

```diff
@@ -552,21 +557,24 @@ func keygenCmd(c *cli.Context) error {
 	return nil
 }
 
-func groupOut(c *cli.Context, group *key.Group) {
+func groupOut(c *cli.Context, group *key.Group, onlyHash bool) {
 	if c.IsSet("out") {
 		groupPath := c.String("out")
 		if err := key.Save(groupPath, group, false); err != nil {
 			fatal("drand: can't save group to specified file name: %v", err)
 		}
 	} else {
-		var buff bytes.Buffer
-		if err := toml.NewEncoder(&buff).Encode(group.TOML()); err != nil {
-			fatal("drand: can't encode group to TOML: %v", err)
+		if !onlyHash {
+			var buff bytes.Buffer
+			if err := toml.NewEncoder(&buff).Encode(group.TOML()); err != nil {
+				fatal("drand: can't encode group to TOML: %v", err)
+			}
+			buff.WriteString("\n")
+			fmt.Printf("Copy the following snippet into a new group.toml file\n")
+			fmt.Printf(buff.String())
 		}
-		buff.WriteString("\n")
-		fmt.Printf("Copy the following snippet into a new group.toml file\n")
-		fmt.Printf(buff.String())
 	}
+	fmt.Printf("\nHash of the group configuration: %x\n", group.Hash())
```

Comment:
can we just print the hash in this mode (not the explanatory string before) so that it can be easily assigned to a bash variable and used in other api commands?